data/json/recipes/food/brewing.json

```diff
@@ -522,5 +522,27 @@
     "qualities": [ { "id": "BOIL", "level": 2 } ],
     "tools": [ [ [ "surface_heat", 6, "LIST" ] ] ],
     "components": [ [ [ "milk_standard_raw", 9, "LIST" ] ], [ [ "cabbage", 1 ], [ "chili_pepper", 1 ], [ "yoghurt", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "brew_rootbeer",
+    "result_mult": 7,
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_BREW",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "time": "20 m",
+    "batch_time_factors": [ 95, 4 ],
+    "book_learn": [ [ "brewing_cookbook", 2 ], [ "family_cookbook", 2 ] ],
+    "proficiencies": [ { "proficiency": "prof_food_prep" }, { "proficiency": "prof_fermenting" } ],
+    "qualities": [ { "id": "CUT", "level": 2 }, { "id": "BOIL", "level": 2 } ],
+    "components": [
+      [ [ "water_clean", 2 ] ],
```

Comment:
Uses 500 ml of water to produce 250 ml of brew. Probably fine since some of it is expected to boil off, and there's not a lot of granularity to liquids.

Reply:
Thinking about this though, if you bulk create this recipe you will potentially use, say, 12.5 liters of water to create 6.25 liters of brew. Might be best to let the water consumption and brew produced basically line up and ignore any boil-off.

Reply:
Yeah, I think cutting in half is a lot, I think I agree that it would be best to ignore boil-off. Was that because of the BOIL level 2 or just a weight problem?

Reply:
I believe boil level 2 refers to the kind of container required to perform the recipe, if the player doesn't have an item with the "boil" quality of a sufficient level they can't craft the recipe. The reason the recipe uses 500 ml at the moment is because it consumes two units of "water_clean" which is a liquid defined elsewhere (1 unit = 250 ml).